12LessonTwelve/homework/index.html

```diff
@@ -1,10 +1,97 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+
+    <title>Lesson 12 | Homework</title>
+    <link rel="stylesheet" href="css/bootstrap.min.css" />
+    <link rel="stylesheet" href="css/style.css" />
```

Comment:
your javascript link should be at the end of the body, just before you close the body tag.